core/pom.xml

```diff
@@ -60,6 +60,7 @@
         <dependency>
             <groupId>com.hierynomus</groupId>
             <artifactId>sshj</artifactId>
+            <version>${sshj.version}</version>
```

Comment:
Do we really need this line? It's defined int he `dependencyManagement` section of `parent/pom.xml`, so we shouldn't have to repeat it here.